### PR TITLE
Add 'tool esp stacktrace' to toit.toit.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: tools/stacktrace.toit [hunk 1]
```diff
@@ -18,36 +18,39 @@ import host.file
...
 main args/List:
-  parsed := null
-  parser := cli.Command "stacktrace"
-      --help=USAGE
+  cmd := build-command
+  cmd.run args
+
+build-command -> cli.Command:
+  cmd := cli.Command "stacktrace"
+      --help="Decode an esp-idf backtrace message from the UART console."
```
Comment (kasperl) [inline tools/stacktrace.toit:31]: esp-idf or ESP-IDF?

File: tools/toit.toit [hunk 2]
```diff
@@ -466,6 +467,12 @@ main args/List:
       --run=:: run-lsp-server it
   tool-command.add tool-lsp-command
 
+  esp-command := cli.Command "esp"
+      --help="ESP-IDF related commands."
```
Comment (kasperl) [inline tools/toit.toit:471]: You want ESP-IDF or esp-idf?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: tools/stacktrace.toit (hunk 1 above)
Comment (floitsch) [reply tools/stacktrace.toit]: ESP-IDF (that's what they use).
done.

File: tools/toit.toit (hunk 2 above)
Comment (floitsch) [reply tools/toit.toit:471]: ESP-IDF.

**Pull request merged**